### ISSUE TITLE
Can we train with customize dataset over the own trained model?

### ISSUE THREAD

**Issue by mmreza79** [NVIDIA/pix2pixHD#80]
I already trained my model with "--netG global" by my own dataset and getting good result using pix2pixHD PyTorch implementation. I would like to train further my existing model with another dataset to see if I achieve better accuracy before train "--netG local". Will my idea work? Can I train further with the new dataset over the existing trained model?

**Comment by tcwang0509** [comment]
Sure you can always finetune your model to see if it achieves good result.

**Comment by abhigoku10** [comment]
@mmreza79 hi how many number of images did u use to train for ur custom dataset

**Comment by mmreza79** [comment]
@abhigoku10, initially I trained with a small dataset with around 2000 images to fine-tune parameters. Finally, I trained with two different datasets with around 28000 and 45000 images.

**Comment by abhigoku10** [comment]
@mmreza79  Thanks for the response , when u trained with 2000 images how was the output? . can you pls share your training params which you used for training ? this will help me to understand my training

**Comment by mmreza79** [comment]
@abhigoku10, Thanks for your interest. Initially I did not get any expected output but I tried to generate ground truth in different ways and fine tune params which gave me expected output. To train model with deep learning is highly depend on how you are preprocessing your ground truth and images. Then you need to fine tune parameters and check which values works better for your dataset.  

You can see #51 where I discussed train and test params. If you share your training commend then it would be better to give you suggestion. Thanks.

**Comment by abhigoku10** [comment]
@mmreza79  thanks for the pointers , shall look into the reference which you have shared and will share the training command in due time
